nservicebus/azure-service-bus/performance-tuning.md

```diff
@@ -10,12 +10,15 @@ tags:
 
 ### CPU vs IO bound processing
 
-There are several things to consider:
+The following settings are used in order to tune performance in Azure Service Bus:
 
+ - `MaximumConcurrencyLevel`
  - `BatchSize`
  - `LockDuration`
  - `MaxDeliveryCount`
 
-In scenarios where handlers are CPU intense and have very little IO, it is advised to lower number of threads to one and have a bigger `BatchSize`. `LockDuration` and `MaxDeliveryCount` might require an adjustment to match the batch size taking in account number of messages that end up in the dead letter queue.
+In scenarios where handlers are CPU intense and have very little IO, it is advised to lower the number of threads to one and increase the `BatchSize`. `LockDuration` and `MaxDeliveryCount` might require an adjustment to match the batch size, taking into account the number of messages that end up in the dead letter queue. 
 
-In scenario where handlers are IO intense, it is advised to set number of threads ([`MaximumConcurrencyLevel`](/nservicebus/operations/tuning.md) in NServiceBus) to 12 threads per logical core and `BatchSize` to a number of messages that takes to process, taking into account possible/measured single message processing time and IO latency. Start with a small `BatchSize` and through adjustment and measurement bring it up, adjusting accordingly `LockDuration` and `MaxDeliveryCount`.
\ No newline at end of file
+In scenario where handlers are IO intense, it is advised to set the number of threads  to 12 threads per logical core using `MaximumConcurrencyLevel` setting, and set the `BatchSize` to a number of messages that takes to process. Take into account the expected (or measured) processing time and IO latency of a single message. Start with a small `BatchSize` and through adjustment and measurement gradually increase it, while adjusting accordingly `LockDuration` and `MaxDeliveryCount`.
```

Comment:
`MaximumConcurrencyLevel` is v5 looks like. In v6 concurrency level is set via code only. Perhaps to include the link we used to have? http://docs.particular.net/nservicebus/operations/tuning